kickeststats/helpers/data.py

```diff
@@ -0,0 +1,10 @@
+from itertools import zip_longest
+from typing import Iterable, Any
+
+
+def grouper(iterable: Iterable, n: int, fillvalue: Any = None) -> Iterable:
```

Comment:
classic grouper :)

Reply:
yes! itertools recipe! why don't they put the recipes in the package? why? :(